docker/docker-compose.yml

```diff
@@ -1,12 +1,12 @@
-version: '3'
+version: '3.5'
 networks:
   default:
     external: true
     name: hermez
 services:
     blockscout:
         container_name: blockscout
-        image: hermez-node-blockscout:latest
+        image: hermez-node-blockscout-dev:latest
```

Comment:
It's not a good practice to have this `-dev` suffix to the image. You should be able to generate the same image in both ways (dev or prod).

Reply:
Well, it's not the same image. You can not run the service in _development_ using the image from _production_ because of the missing `selfsigned_key.pem`. This incompatibility was my reasoning for the change.

In the end, this docker-compose is only here for dev purposes, why is it not a good practice to have this `-dev` suffix to the image?

Reply:
Done as agreed on the meet.